server.js

```diff
@@ -21,3 +22,12 @@ app.listen(PORT, function(){
 //     headers: {Authorization: `token ${process.env.GITHUB_TOKEN}`}
 //   }))(request, response);
 // }
+function proxyGitHub(request, response) {
```

Comment:
I would like to also say you can totally keep it this way, since you will need all of this code(including requestProxy) for your Heroku deployment. So you can keep it this way and work out the bugs or switch it to a simpler solution, then work through this. Either way works(let me know if I have confused you, even after project week :)